pkg/plugin/config/config_suite_test.go

```diff
@@ -4,10 +4,12 @@ import (
 	"testing"
 
 	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
 	. "github.com/onsi/gomega"
 )
 
 func TestConfig(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Plugin Config Loader Suite")
+	junitReporter := reporters.NewJUnitReporter("junit.xml")
+	RunSpecsWithDefaultAndCustomReporters(t, "Plugin Config Loader Suite", []Reporter{junitReporter})
```

Comment:
Can we move it to `internal/test` package and re-use across all the suites instead?